### PR TITLE
Adding initial React scripts/project structure

### PR DESCRIPTION
**Pull request by leorod**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2229 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```
Comment (fpiagent) [inline README.md]: Maybe eventually we want to remove this README and have our own sexy and small readme.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,2229 @@
...
+
+it('renders welcome message', () => {
+  const wrapper = shallow(<App />);
+  const welcome = <h2>Welcome to React</h2>;
+  // expect(wrapper.contains(welcome)).to.equal(true);
+  expect(wrapper.contains(welcome)).toEqual(true);
+});
+```
+
+All Jest matchers are [extensively documented here](http://face
```
Comment (fpiagent) [inline README.md]: Gheez louise! THIS IS a long Readme.md

File: src/App.test.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
```
Comment (fpiagent) [inline src/App.test.js:5]: Integration tests! nice

**Review by fpiagent — APPROVED** [review]
LGTM
Just check if you want to merge the README.md

File: README.md (hunk 1 above)
Comment (leorod) [reply README.md]: Totally agree, it's the one that create-react-app generates automagically. Lemme kill that bastard

**Pull request merged**